environment.yml

```diff
@@ -13,12 +13,11 @@ dependencies:
   - cdo>=1.9.7
   - imagemagick
   - nco
+  - scikit-learn  # install from same installer as scipy
 
   # Multi language support:
   - python>=3.6
-  - libunwind  # Needed for Python3.7+
-  - ncl>=6.5.0
```

Comment:
I believe there was some problem with older versions of NCL, so it might be better to keep this

Reply:
it will always install the latest `ncl` now, if it reverts back to that ancient version I'll eat my hat.. It's a bit more relaxed for conda if you don't pin versions. We can probably write an environment test to check versions on the CI machine, that'd be a more elegant solution than pinning stuff :beer:

Reply:
> it will always install the latest ncl now

While that may have been the case when you tried, it may not always be true. Requiring a minimum version is not that restrictive and anyway, we should not give conda the freedom to create an environment that includes a version that is so outdated that it does not work.

Reply:
okidokey, I repinned lower bound :beer:

Reply:
Thanks!